src/test/java/org/opendevstack/provision/controller/DefaultControllerTest.java

```diff
@@ -44,35 +45,31 @@
 import org.springframework.web.context.WebApplicationContext;
 
 @SpringBootTest
-@ActiveProfiles({"crowd", "utestcrowd", "quickstarters"})
+@ActiveProfiles({"crowd", "quickstarters"})
 public class DefaultControllerTest {
 
   @MockBean private IJobExecutionAdapter jobExecutionAdapter;
 
   @MockBean private StorageAdapter storageAdapter;
 
-  @MockBean private CrowdAuthenticationManager crowdAuthenticationManager;
+  @Mock private CrowdAuthenticationManager crowdAuthenticationManager;
```

Comment:
Same as above, I think we should go for `@MockBean` instead of `@Mock` to not mix mocking utilities.

Reply:
ok in this case I changed it back for you for now ;)